jira/resources.py

```diff
@@ -692,6 +692,27 @@ def update(  # type: ignore[override] # incompatible supertype ignored
 
         super(Issue, self).update(async_=async_, jira=jira, notify=notify, fields=data)
 
+    def get_field(self, field_name: str) -> Any:
+        """Obtain the (parsed) value from the Issue's field.
+
+        Args:
+            fieldName (str): The name of the field to get
+
+        Raises:
+            AttributeError: If the field does not exist or if the field starts with an ``_``
+
+        Returns:
+            Any: Returns the parsed data stored in the field. For example, "project" would be of class :py:class:`Project`
+        """
+
+        if fieldName.startswith("_"):
+            raise AttributeError(
+                f"An issue field_name cannot start with underscore (_): {fieldName}",
+                fieldName,
+            )
+        else:
+            return getattr(self.fields, fieldName)
```

Comment:
@dvaerum , yep need to use the new variable name in the function body (and docstring) too:

```suggestion
            field_name (str): The name of the field to get

        Raises:
            AttributeError: If the field does not exist or if the field starts with an ``_``

        Returns:
            Any: Returns the parsed data stored in the field. For example, "project" would be of class :py:class:`Project`
        """

        if field_name.startswith("_"):
            raise AttributeError(
                f"An issue field_name cannot start with underscore (_): {field_name}",
                field_name,
            )
        else:
            return getattr(self.fields, field_name)
```

Reply:
aaah, so because one fails the rest gets canceled, makes sense, I don't know why I didn't look in the one there was red :joy: